frontend/components/CustomLink/_styles.scss

```
@@ -1,36 +1,19 @@
 .custom-link {
   @include link;
+  font-weight: $regular;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)